### DISCUSSION TITLE
Formatação de perfil

### DISCUSSION THREAD

**Discussion by YasminCury** [community/community#137573] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Existe algum material que ensine Markdown para personalizar o perfil?

**Comment by AIExxplorer** [comment, marked as answer]
Sim, você pode acessar o livro diretamente no meu repositório ACERVO DE TECNOLOGIA, lá contém este material disponível:

https://github.com/AIExxplorer/ACERVO_DE_TECNOLOGIA/tree/main/TI_e_Desenvolvimento/Markdown

E de vários outros temas!

**Comment by LiteBrite82** [comment]
Hi @YasminCury , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
